src/mpl/configure.ac

```diff
@@ -962,6 +962,18 @@ if test "X${have_cuda}" = "Xyes" ; then
 fi
 AM_CONDITIONAL([MPL_HAVE_CUDA],[test "X${have_cuda}" = "Xyes"])
 
+
+PAC_SET_HEADER_LIB_PATH([ze])
+PAC_SET_HEADER_LIB_PATH([opencl])
+PAC_CHECK_HEADER_LIB([level_zero/ze_api.h],[ze_loader],[zeInit],[have_ze=yes],[have_ze=no])
+# ze has dependency on opencl
+AC_CHECK_HEADERS([CL/cl.h], [have_cl=yes], [have_cl=no])
```

Comment:
Can you check for a library function too, and not just a header?

Reply:
Sure

Reply:
Done